app/views/posts/index.html.erb

```diff
@@ -1,12 +1,20 @@
 <%= render 'shared/user', user: @user%>
-<ul>
+<ul class='all-posts'>
   <% if @posts.empty?%>
     <p class='post-card-no-click'>No posts</p>
   <% else%>
-  <div class='users-bio'><%=@user.bio%></div>
   <% @posts.each do |post|%>
     <li>
-      <%= render 'shared/post', post: post%>
+      <div class='posts-card-div'><%= render 'shared/post', post: post%></div>
+      <div class='posts-comments'>
+        <h3>Comments:</h3>
+        <% post.recent_5_comments.each do |comment,index| %>
```

Comment:
- [ OPTIONAL ] After fixing the errors, I noticed that on your `Posts Show Page` your app continues to show only 5 recent comments instead of all the comments so I will suggest you fix that as well :wink: 
- Your App
![image](https://github.com/lorenzoworx/velvetScribe/assets/102699958/d7ee77de-1131-4fa3-b4a3-6a47d56465a7)
- The Given WireFrame 
![image](https://github.com/lorenzoworx/velvetScribe/assets/102699958/8b921272-f26f-4b5a-a6fb-142a4b6644b3)